docs/tutorials/reusing-ssh-connections.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Reusing SSH Connections
+group: advanced-networking
+author: [ "@Dave-Baum" ]
+---
+
+* Table of Contents
+{:toc}
+
+## Intro
+
+Establishing an ssh connection requires a a lot of communication between your host computer
+and the EV3 and can take several seconds. Although this is fine a long-lived connection,
+it can be quite tedious if you want to use ssh to invoke a single command or scp to transfer
+files. By instructing ssh to reuse connections, each connection after the first one will
+reuse the session that was already established, drastically speeding up ssh and scp commands.
+
+## Configuration (macOS, Linux)
+
+Edit or create a new file at `~/.ssh/config`.  Add the following contents:
+
+    Host ev3
+    HostName ev3dev.local.
+    User robot
+    ControlMaster auto
+    ControlPath ~/.ssh/master-%r@%h:%p
+
+This configuration does several things: it establishes `ev3` as a nickname (so you don't have to type
+`ev3dev.local.` all the time), defaults to `robot` as the user name, and  instructs ssh to automatically
```

Comment:
It actually works both ways. The trailing dot means that it is an absolute domain name rather than relative. It doesn't really matter for this tutorial, so we can leave it as-is (as of a814064).